cmd/ddev/cmd/delete-images.go

```diff
@@ -65,40 +88,39 @@ ddev delete images -y`,
 			for _, tag := range image.RepoTags {
 				// If a webimage, but doesn't match our webimage, delete it
 				if strings.HasPrefix(tag, version.WebImg) && !strings.HasPrefix(tag, webimg) && !strings.HasPrefix(tag, webimg+"-built") {
-					if err = dockerutil.RemoveImage(tag); err != nil {
-						util.Warning("Failed to remove %s: %v", tag, err)
-					}
+					dockerRemoveImage(tag)
 				}
 				if strings.HasPrefix(tag, "drud/ddev-dbserver") && !strings.HasSuffix(tag, keepDBImageTag) && !strings.HasSuffix(tag, keepDBImageTag+"-built") {
-					if err = dockerutil.RemoveImage(tag); err != nil {
-						util.Warning("Unable to remove %s: %v", tag, err)
-					}
+					dockerRemoveImage(tag)
 				}
 				// If a dbaimage, but doesn't match our dbaimage, delete it
 				if strings.HasPrefix(tag, version.DBAImg) && !strings.HasPrefix(tag, dbaimage) {
-					if err = dockerutil.RemoveImage(tag); err != nil {
-						util.Warning("Failed to remove %s: %v", tag, err)
-					}
+					dockerRemoveImage(tag)
 				}
 				// If a routerImage, but doesn't match our routerimage, delete it
 				if strings.HasPrefix(tag, version.RouterImage) && !strings.HasPrefix(tag, routerimage) {
-					if err = dockerutil.RemoveImage(tag); err != nil {
-						util.Warning("Failed to remove %s: %v", tag, err)
-					}
+					dockerRemoveImage(tag)
 				}
 				// If a sshAgentImage, but doesn't match our sshAgentImage, delete it
 				if strings.HasPrefix(tag, version.SSHAuthImage) && !strings.HasPrefix(tag, sshimage) && !strings.HasPrefix(tag, sshimage+"-built") {
-					if err = dockerutil.RemoveImage(tag); err != nil {
-						util.Warning("Failed to remove %s: %v", tag, err)
-					}
+					dockerRemoveImage(tag)
 				}
 			}
 		}
 		util.Success("Any non-current images discovered were deleted.")
 	},
 }
 
+// dockerRemoveImage Deletes an images by its tag
+func dockerRemoveImage(tag string) {
```

Comment:
I'm not completely sure why this wrapper function is needed...

Reply:
I was trying to reduce duplicated code with this. I will remove it.